chemprop/features/features_generators.py

```diff
@@ -189,7 +173,7 @@ def rdkit_2d_normalized_features_generator(mol_data: Union[Molecule, List[List[M
 
 @register_features_generator('custom')
 def custom_features_generator(mol_data: Union[Molecule, List[List[Molecule]]]) -> np.ndarray:
-    if type(mol_data) == list:
+    if isinstance(mol_data, Iterable) and not isinstance(mol_data, AnyStr):
```

Comment:
```suggestion
    if isinstance(mol_data, Iterable) and not isinstance(mol_data, str):
```

Reply:
Thanks, originally wanted to use AnyStr but PyCharm was linting it as an error. Looks like that's erroneous though: https://youtrack.jetbrains.com/issue/PY-32860

I'll switch them over.

Reply:
Updating my previous comment. You can't use AnyStr in isinstance (throws a runtime error), and the only functionality it adds over str is supporting inputs of byte objects. The MoleculeDatapoint class (and other classes within chemprop) only support str, so I don't see a reason to use AnyStr here over str.